hudi-common/src/main/java/org/apache/hudi/common/table/log/BaseHoodieLogRecordReader.java

```diff
@@ -348,8 +349,21 @@ private void scanInternalV1(Option<KeySpec> keySpecOpt) {
         // Eat exception as we do not want to mask the original exception that can happen
         LOG.error("Unable to close log format reader", ioe);
       }
-      LOG.info("Finished scanning log files. Total log files: {}, Total log blocks: {}, Total rollbacks: {}, Total corrupt blocks: {}",
-          totalLogFiles.get(), totalLogBlocks.get(), totalRollbacks.get(), totalCorruptBlocks.get());
+      if (!logFiles.isEmpty()) {
+        try {
+          String fileId = FSUtils.getFileIdFromLogPath(logFiles.get(0).getPath());
+          LOG.info("Finished scanning log files. FileId: {}, BaseInstantTime: {}, "
```

Comment:
`BaseInstantTime` and `latestInstantTime` do not match.

Reply:
This is not addressed. In table version 9, the instant time extracted from the log file reflect the delta commit time, not the base file instant time, so `BaseInstantTime` is confusing.  I suggest to make it `LogFileInstantTime`.